addons/google_calendar/utils/google_event.py

```diff
@@ -63,7 +63,7 @@ def ids(self):
 
     @property
     def rrule(self):
-        if self.recurrence:
+        with suppress(StopIteration, TypeError):
```

Comment:
Thanks for your contribution. It is welcome !
I am not a fan of this approach. It feels like hiding things under the carpet. 
I prefer to explicitly look for rrule in this rrule property.

I am asked @pedrambiria to understand why https://github.com/odoo/odoo/pull/87697  was closed but I prefer this approach. In your PR description you explain that your approach is an improved version of the fw port. Could you explain your point ? Thanks :-)

Reply:
`self.recurrence` was sometimes `True`. That raises a `TypeError` with your code.

Reply:
Strange. Do you have a reproducible way to get `self.recurrence == True` ?